client/fingerprint_manager.go

```diff
@@ -65,28 +65,28 @@ func (fm *FingerprintManager) getNode() *structs.Node {
 }
 
 // Run starts the process of fingerprinting the node. It does an initial pass,
-// identifying whitelisted and blacklisted fingerprints/drivers. Then, for
+// identifying allowlisted and denylisted fingerprints/drivers. Then, for
 // those which require periotic checking, it starts a periodic process for
 // each.
 func (fp *FingerprintManager) Run() error {
 	// First, set up all fingerprints
 	cfg := fp.getConfig()
-	whitelistFingerprints := cfg.ReadStringListToMap("fingerprint.whitelist")
-	whitelistFingerprintsEnabled := len(whitelistFingerprints) > 0
-	blacklistFingerprints := cfg.ReadStringListToMap("fingerprint.blacklist")
+	allowlistFingerprints := cfg.ReadStringListToMap("fingerprint.allowlist")
+	allowlistFingerprintsEnabled := len(allowlistFingerprints) > 0
+	denylistFingerprints := cfg.ReadStringListToMap("fingerprint.denylist")
```

Comment:
Given that these are user visible configuration, it'd be nice to keep accepting old names (potentially with a warning) for backward compatibility but only document the new keys.